assets/src/features/room-page/components/PageLayout.tsx

```diff
@@ -1,26 +1,27 @@
 import clsx from "clsx";
 import React, { PropsWithChildren } from "react";
+import BlockingScreen from "../../shared/components/BlockingScreen";
+import useHorizontalMobile from "../../shared/hooks/useHorizontalMobile";
 import Navbar from "./Navbar";
 
 const PageLayout: React.FC<PropsWithChildren> = ({ children }) => {
+  const isHorizontalMobile = useHorizontalMobile();
```

Comment:
Maybe it would be nice to change the `useHorizontalMobile()` hook to something more universal like `useCheckScreenType()` that would provide information whether it is a desktop screen, horizontal or vertical mobile screen.
If this is the scope of this task, I'll leave it under your consideration.